src/6.7/6.7.2.0.md

```diff
@@ -0,0 +1,412 @@
+Additionally, the sorting of the properties in the group was improved to use a more performant algorithm. 
+This will improve the performance of loading the property filters by up to 50%, especially with a large number of property filters.
+
+### Efficiency Improvements
+
+#### Lower memory footprint on data imports
+
+Memory usage, especially when importing a lot of custom fields, could be greatly reduced by reusing dynamic field definition objects and more efficient validator constraint caching.
+
+Additionally, a memory leak in HTMLSanitize was fixed, which increased memory usage the more sanitized fields were written in a single request. 
+
+This makes shopware more memory efficient, especially when writing bigger batches of data in a single request.
+
+#### More efficient administration OpenSearch indexing
+
+The indexing for the administration search indices on OpenSearch was improved to only reindex entities when search-relevant data was changed and not directly when any change happened to the entity.
+
+### Return address in documents
+
+The option `Display company` address in the `Company settings` section of the document configuration is now split into `Display return address` and `Display company address`.  
+The former toggles the display of the return address above the customer address in the address block.  
+The latter toggles the display of the company address below the header on the right-hand side of the document.
+
+### Quality of Life Improvement: Administration forwards to the correct path after Login
+
+When you open the administration with a deep link (e.g. to product list or order page), and you'll get redirected to the login first, you will now be forwarded to the original page you wanted to visit after sucessful login instead of always landing on the dashboard after login.
+
+### Configurable minimum search length
+
+You can now define how many characters a search query must have before results are shown. This can help reducing irrelevant searches like for “a” or “an”.
+
+By default, Shopware sets the minimum search term length to 3 characters, which means any query shorter than that will be ignored. Go to **Settings → Shop → Search** and add your minimum number to the field labeled "Minimum characters for search". To ensure the changes take effect, rebuild the search index. This is typically done under **Settings → System → Caches & Indexes → Search** (or Live Search)**.
+
+### More custom field set support in Apps
+
+Apps can now also register custom field sets for the `Unit` and `Newsletter Recipient` entities.
+
+### Better accessibility with ARIA labels
+
+Slider links now include descriptive text that can be read by screen readers, making it easier for people with visual impairments to navigate your content.
+
+### Faster loading with fetchpriority 
+
+This improves the Largest Contentful Paint (LCP), an important metric for PageSpeed and Google’s Core Web Vitals.
```

Comment:
It is challenging to begin with this sentence, while it just has impact on less content.